README.adoc

```diff
@@ -171,8 +171,10 @@ You can build everything from the root of the repository.
 
 `./mvnw clean install`
 
-TIP: By default the build skips long running tests, annotated with `@Tag("integration")`,including tests requiring Testcontainers. To run the full test suite, activate the `integration` Maven profile:
-`./mvnw clean install -Pintegration`. This applies to building functions or individual applications as well.
+TIP: By default, the build skips long-running tests, annotated with `@Tag("integration")`,including tests requiring Testcontainers.
```

Comment:
**[UNRELATED]** I was in the README and noticed the separate-line-per-sentence was not being followed in this location.

Reply:
I backed off this change as it was in conflict. Will add in separate PR.